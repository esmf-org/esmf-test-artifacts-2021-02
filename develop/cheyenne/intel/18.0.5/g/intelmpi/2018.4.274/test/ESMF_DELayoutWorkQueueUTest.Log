20201005 052956.923 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20201005 052956.923 INFO             PET0 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20201005 052956.923 INFO             PET0 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20201005 052956.923 INFO             PET0 !!! FOR PRODUCTION RUNS, USE:                      !!!
20201005 052956.923 INFO             PET0 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20201005 052956.923 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20201005 052956.923 INFO             PET0 Running with ESMF Version   : ESMF_8_1_0_beta_snapshot_31-7-g0dc67b5b55
20201005 052956.923 INFO             PET0 ESMF library build date/time: "Oct  5 2020" "04:44:32"
20201005 052956.923 INFO             PET0 ESMF library build location : /glade/scratch/himanshu/scripts_dir/daily/intel_cheyenne/esmf
20201005 052956.923 INFO             PET0 ESMF_COMM                   : intelmpi
20201005 052956.923 INFO             PET0 ESMF_NETCDF                 : disabled
20201005 052956.923 INFO             PET0 ESMF_PNETCDF                : disabled
20201005 052956.923 INFO             PET0 ESMF_PIO                    : enabled
20201005 052956.923 INFO             PET0 ESMF_YAMLCPP                : enabled
20201005 052956.923 INFO             PET0 ESMF_MOAB                   : enabled
20201005 052956.924 INFO             PET0  Beginning Test, file ESMF_DELayoutWorkQueueUTest.F90, line 170
20201005 052956.924 INFO             PET0  NUMBER_OF_PROCESSORS           4
20201005 052956.924 INFO             PET0  PASS  GridCompCreate() - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 187
20201005 052956.924 INFO             PET0  PASS  GridCompSetServices() - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 193
20201005 052956.925 INFO             PET0  PASS  Run work queue - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 201
20201005 052956.925 INFO             PET0  PASS  GridCompDestroy() - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 209
20201005 053002.952 INFO             PET0  PASS  GridCompCreate() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 217
20201005 053002.953 INFO             PET0  PASS  GridCompSetVM() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 227
20201005 053002.953 INFO             PET0 service: 0  PTHREAD_STACK_MIN: 16384 bytes  stack_size: 20971520 bytes
20201005 053002.953 INFO             PET0 service: 0  PTHREAD_STACK_MIN: 16384 bytes  stack_size: 20971520 bytes
20201005 053002.953 INFO             PET0 service: 0  PTHREAD_STACK_MIN: 16384 bytes  stack_size: 20971520 bytes
20201005 053002.953 INFO             PET0 service: 0  PTHREAD_STACK_MIN: 16384 bytes  stack_size: 20971520 bytes
20201005 053002.954 INFO             PET0  PASS  GridCompSetServices() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 233
20201005 053005.496 INFO             PET0  PASS  Run work queue - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 241
20201005 053005.496 INFO             PET0  PASS  GridCompDestroy() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 249
20201005 053005.496 INFO             PET0  Ending Test, file ESMF_DELayoutWorkQueueUTest.F90, line 257
20201005 053005.496 INFO             PET0 Finalizing ESMF
20201005 052956.923 INFO             PET1 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20201005 052956.923 INFO             PET1 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20201005 052956.923 INFO             PET1 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20201005 052956.923 INFO             PET1 !!! FOR PRODUCTION RUNS, USE:                      !!!
20201005 052956.923 INFO             PET1 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20201005 052956.923 INFO             PET1 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20201005 052956.923 INFO             PET1 Running with ESMF Version   : ESMF_8_1_0_beta_snapshot_31-7-g0dc67b5b55
20201005 052956.923 INFO             PET1 ESMF library build date/time: "Oct  5 2020" "04:44:32"
20201005 052956.923 INFO             PET1 ESMF library build location : /glade/scratch/himanshu/scripts_dir/daily/intel_cheyenne/esmf
20201005 052956.923 INFO             PET1 ESMF_COMM                   : intelmpi
20201005 052956.923 INFO             PET1 ESMF_NETCDF                 : disabled
20201005 052956.923 INFO             PET1 ESMF_PNETCDF                : disabled
20201005 052956.923 INFO             PET1 ESMF_PIO                    : enabled
20201005 052956.923 INFO             PET1 ESMF_YAMLCPP                : enabled
20201005 052956.923 INFO             PET1 ESMF_MOAB                   : enabled
20201005 052956.924 INFO             PET1  Beginning Test, file ESMF_DELayoutWorkQueueUTest.F90, line 170
20201005 052956.924 INFO             PET1  NUMBER_OF_PROCESSORS           4
20201005 052956.924 INFO             PET1  PASS  GridCompCreate() - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 187
20201005 052956.924 INFO             PET1  PASS  GridCompSetServices() - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 193
20201005 053000.952 INFO             PET1  PASS  Run work queue - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 201
20201005 053000.952 INFO             PET1  PASS  GridCompDestroy() - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 209
20201005 053002.952 INFO             PET1  PASS  GridCompCreate() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 217
20201005 053002.953 INFO             PET1  PASS  GridCompSetVM() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 227
20201005 053002.953 INFO             PET1 service: 1  PTHREAD_STACK_MIN: 16384 bytes  stack_size: 4194304 bytes
20201005 053002.953 INFO             PET1 service: 1  PTHREAD_STACK_MIN: 16384 bytes  stack_size: 4194304 bytes
20201005 053002.954 INFO             PET1  PASS  GridCompSetServices() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 233
20201005 053005.461 INFO             PET1  PASS  Run work queue - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 241
20201005 053005.461 INFO             PET1  PASS  GridCompDestroy() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 249
20201005 053005.461 INFO             PET1  Ending Test, file ESMF_DELayoutWorkQueueUTest.F90, line 257
20201005 053005.461 INFO             PET1 Finalizing ESMF
20201005 052956.923 INFO             PET2 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20201005 052956.923 INFO             PET2 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20201005 052956.923 INFO             PET2 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20201005 052956.923 INFO             PET2 !!! FOR PRODUCTION RUNS, USE:                      !!!
20201005 052956.923 INFO             PET2 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20201005 052956.923 INFO             PET2 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20201005 052956.923 INFO             PET2 Running with ESMF Version   : ESMF_8_1_0_beta_snapshot_31-7-g0dc67b5b55
20201005 052956.923 INFO             PET2 ESMF library build date/time: "Oct  5 2020" "04:44:32"
20201005 052956.923 INFO             PET2 ESMF library build location : /glade/scratch/himanshu/scripts_dir/daily/intel_cheyenne/esmf
20201005 052956.923 INFO             PET2 ESMF_COMM                   : intelmpi
20201005 052956.923 INFO             PET2 ESMF_NETCDF                 : disabled
20201005 052956.923 INFO             PET2 ESMF_PNETCDF                : disabled
20201005 052956.923 INFO             PET2 ESMF_PIO                    : enabled
20201005 052956.923 INFO             PET2 ESMF_YAMLCPP                : enabled
20201005 052956.923 INFO             PET2 ESMF_MOAB                   : enabled
20201005 052956.924 INFO             PET2  Beginning Test, file ESMF_DELayoutWorkQueueUTest.F90, line 170
20201005 052956.924 INFO             PET2  NUMBER_OF_PROCESSORS           4
20201005 052956.924 INFO             PET2  PASS  GridCompCreate() - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 187
20201005 052956.924 INFO             PET2  PASS  GridCompSetServices() - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 193
20201005 053002.952 INFO             PET2  PASS  Run work queue - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 201
20201005 053002.952 INFO             PET2  PASS  GridCompDestroy() - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 209
20201005 053002.952 INFO             PET2  PASS  GridCompCreate() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 217
20201005 053002.952 INFO             PET2  PASS  GridCompSetVM() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 227
20201005 053002.953 INFO             PET2 service: 1  PTHREAD_STACK_MIN: 16384 bytes  stack_size: 4194304 bytes
20201005 053002.953 INFO             PET2 service: 1  PTHREAD_STACK_MIN: 16384 bytes  stack_size: 4194304 bytes
20201005 053002.954 INFO             PET2  PASS  GridCompSetServices() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 233
20201005 053005.456 INFO             PET2  PASS  Run work queue - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 241
20201005 053005.457 INFO             PET2  PASS  GridCompDestroy() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 249
20201005 053005.457 INFO             PET2  Ending Test, file ESMF_DELayoutWorkQueueUTest.F90, line 257
20201005 053005.457 INFO             PET2 Finalizing ESMF
20201005 052956.923 INFO             PET3 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20201005 052956.923 INFO             PET3 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20201005 052956.923 INFO             PET3 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20201005 052956.923 INFO             PET3 !!! FOR PRODUCTION RUNS, USE:                      !!!
20201005 052956.923 INFO             PET3 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20201005 052956.923 INFO             PET3 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20201005 052956.923 INFO             PET3 Running with ESMF Version   : ESMF_8_1_0_beta_snapshot_31-7-g0dc67b5b55
20201005 052956.923 INFO             PET3 ESMF library build date/time: "Oct  5 2020" "04:44:32"
20201005 052956.923 INFO             PET3 ESMF library build location : /glade/scratch/himanshu/scripts_dir/daily/intel_cheyenne/esmf
20201005 052956.923 INFO             PET3 ESMF_COMM                   : intelmpi
20201005 052956.923 INFO             PET3 ESMF_NETCDF                 : disabled
20201005 052956.923 INFO             PET3 ESMF_PNETCDF                : disabled
20201005 052956.923 INFO             PET3 ESMF_PIO                    : enabled
20201005 052956.923 INFO             PET3 ESMF_YAMLCPP                : enabled
20201005 052956.923 INFO             PET3 ESMF_MOAB                   : enabled
20201005 052956.924 INFO             PET3  Beginning Test, file ESMF_DELayoutWorkQueueUTest.F90, line 170
20201005 052956.924 INFO             PET3  NUMBER_OF_PROCESSORS           4
20201005 052956.924 INFO             PET3  PASS  GridCompCreate() - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 187
20201005 052956.924 INFO             PET3  PASS  GridCompSetServices() - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 193
20201005 052956.925 INFO             PET3  PASS  Run work queue - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 201
20201005 052956.925 INFO             PET3  PASS  GridCompDestroy() - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 209
20201005 053002.952 INFO             PET3  PASS  GridCompCreate() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 217
20201005 053002.952 INFO             PET3  PASS  GridCompSetVM() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 227
20201005 053002.953 INFO             PET3 service: 1  PTHREAD_STACK_MIN: 16384 bytes  stack_size: 4194304 bytes
20201005 053002.953 INFO             PET3 service: 1  PTHREAD_STACK_MIN: 16384 bytes  stack_size: 4194304 bytes
20201005 053002.954 INFO             PET3  PASS  GridCompSetServices() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 233
20201005 053005.496 INFO             PET3  PASS  Run work queue - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 241
20201005 053005.496 INFO             PET3  PASS  GridCompDestroy() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 249
20201005 053005.496 INFO             PET3  Ending Test, file ESMF_DELayoutWorkQueueUTest.F90, line 257
20201005 053005.496 INFO             PET3 Finalizing ESMF
