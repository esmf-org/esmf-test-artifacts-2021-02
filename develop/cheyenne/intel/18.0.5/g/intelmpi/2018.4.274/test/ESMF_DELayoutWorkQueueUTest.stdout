  Beginning Test, file ESMF_DELayoutWorkQueueUTest.F90, line 170
  Beginning Test, file ESMF_DELayoutWorkQueueUTest.F90, line 170
  Beginning Test, file ESMF_DELayoutWorkQueueUTest.F90, line 170
  Beginning Test, file ESMF_DELayoutWorkQueueUTest.F90, line 170
  NUMBER_OF_PROCESSORS           4
  NUMBER_OF_PROCESSORS           4
  NUMBER_OF_PROCESSORS           4
  NUMBER_OF_PROCESSORS           4
  PASS  GridCompCreate() - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 187
  PASS  GridCompCreate() - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 187
  PASS  GridCompCreate() - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 187
  PASS  GridCompCreate() - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 187
  PASS  GridCompSetServices() - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 1
 93
  PASS  GridCompSetServices() - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 1
 93
  PASS  GridCompSetServices() - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 1
 93
  PASS  GridCompSetServices() - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 1
 93
  PASS  Run work queue - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 201
  PASS  Run work queue - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 201
 <round 1> PET            0  time:   2.558231353759766E-004
 <round 1> PET            3  time:   2.579689025878906E-004
  PASS  GridCompDestroy() - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 209
  PASS  GridCompDestroy() - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 209
  PASS  Run work queue - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 201
 <round 1> PET            1  time:    4.02683901786804     
  PASS  GridCompDestroy() - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 209
  PASS  Run work queue - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 201
 <round 1> PET            2  time:    6.02697587013245     
  PASS  GridCompDestroy() - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 209
  PASS  GridCompCreate() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 217
  PASS  GridCompCreate() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 217
  PASS  GridCompCreate() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 217
  PASS  GridCompCreate() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 217
 *** hi from mygcomp_setvm_withthreads ***
 *** hi from mygcomp_setvm_withthreads ***
  PASS  GridCompSetVM() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 227
 *** hi from mygcomp_setvm_withthreads ***
 *** hi from mygcomp_setvm_withthreads ***
  PASS  GridCompSetVM() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 227
  PASS  GridCompSetVM() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 227
  PASS  GridCompSetVM() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 227
  PASS  GridCompSetServices() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 2
 33
  PASS  GridCompSetServices() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 2
 33
  PASS  GridCompSetServices() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 2
 33
  PASS  GridCompSetServices() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 2
 33
  PASS  Run work queue - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 241
 <round 2> PET            2  time:    2.50510287284851     
  PASS  GridCompDestroy() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 249
  Ending Test, file ESMF_DELayoutWorkQueueUTest.F90, line 257
  PASS  Run work queue - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 241
 <round 2> PET            3  time:    2.51594090461731     
  PASS  GridCompDestroy() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 249
  Ending Test, file ESMF_DELayoutWorkQueueUTest.F90, line 257
  PASS  Run work queue - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 241
  PASS  Run work queue - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 241
 <round 2> PET            1  time:    2.55595302581787     
 <round 2> PET            0  time:    2.55596303939819     
  PASS  GridCompDestroy() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 249
  Ending Test, file ESMF_DELayoutWorkQueueUTest.F90, line 257
  PASS  GridCompDestroy() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 249
  Ending Test, file ESMF_DELayoutWorkQueueUTest.F90, line 257
  PET           2  Test Elapsed Time    6124.54300000000       msec.
  PET           3  Test Elapsed Time    6111.62500000000       msec.
  PET           0  Test Elapsed Time    16056.7060000000       msec.
  PET           1  Test Elapsed Time    6072.11200000000       msec.
Sun Dec 13 18:30:54 MST 2020
